docs: background backup FAQ (#9142)
immich-app/immich: docs/docs/FAQ.mdx

Sometimes Immich logs errors such as "duplicate key value violates unique constraint" or "column (...) of relation (...) already exists". Because of Immich's container structure, this error can be seen when both immich and immich-microservices start at the same time and attempt to migrate or create the database structure. Since the database migration is run sequentially and inside of transactions, this error message does not cause harm to your installation of Immich and can safely be ignored. If needed, you can manually restart Immich by running `docker restart immich immich-microservices`.

### Why does foreground backup stop when I navigate away from the app? Shouldn't it transfer the job to background backup?

Foreground backup and background backup are two separate mechanisms. They don't communicate or interact with each other.

> Foreground backup is controlled by the user's action, while background backup is controlled by your phone's operating system. When the app is put in the background, the invocation of background tasks is delegated to the phone's operating system scheduler. It makes the call when the background task can run and how long it can run.

Reviewer: ```suggestion
Foreground backup is controlled by the user's action, while background backup is controlled by your device's operating system. When the app is put in the background, the invocation of background tasks is delegated to the device's operating system scheduler. It makes the call when the background task can run and how long it can run.
```

> The behaviors differ depending on your phone's manufacturer; most are under battery-saving policies.

Reviewer: ```suggestion
The behaviors differ based on your device manufacturer and operating system, but most are related to battery-saving policies.
```

> The behaviors differ based on your device manufacturer and operating system, but most are related to battery-saving policies.

Reviewer 2: ```suggestion
The behavior differs based on your device manufacturer and operating system, but most are related to battery-saving policies.
```

### Why is background backup on iOS not working?

> iPhone determines if a particular app can invoke background tasks by multiple factors, most of which we don't know and don't have control over. To increase the likelihood that background backup can run, follow the steps below:

Reviewer: ```suggestion
On iOS (iPhone and iPad), the operating system determines if a particular app can invoke background tasks based on multiple factors, most of which the Immich app has no control over. To increase the likelihood that the background backup task is run, follow the steps below:
```

> - Make sure you enable Background App Refresh in the phone's settings. `Settings > General > Background App Refresh`.

Reviewer: ```suggestion
- Enable Background App Refresh for Immich in the iOS settings at `Settings > General > Background App Refresh`.
```

> - Disable Background App Refresh for apps that don't need background tasks to run. This will reduce the competition for background task invocation for Immich.

Reviewer: ```suggestion
- Disable Background App Refresh for apps that don't need background tasks to run.
  - This will reduce the competition for background task invocation for Immich.
```

Reviewer 2: I personally like the current formatting more

Reviewer: Makes sense. I was trying to divide "action steps" from "background details", but it's not strictly necessary for a short blurb like this.
